internal/http/services/owncloud/ocdav/copy.go

```diff
@@ -303,6 +303,8 @@ func (s *svc) executePathCopy(ctx context.Context, selector pool.Selectable[gate
 			return err
 		}
 		httpUploadReq.Header.Set(datagateway.TokenTransportHeader, uploadToken)
+		httpUploadReq.Header.Set(net.HeaderUploadLength, strconv.FormatUint(cp.sourceInfo.GetSize(), 10))
```

Comment:
I think the `httpUploadReq.Header.Set(net.HeaderUploadLength...` line is unnecessary as we are using the `simple` PUT, not tus.
```suggestion
```